### PR TITLE
Fix relations consistency

### PR DESCRIPTION
**Pull request by michitaro**
This PR fixes some inconsistencies in model relations detected by [the automated test](https://github.com/Subaru-PFS/spt_operational_database/pull/38).

@kiyoyabe Could you review them? I'm not sure if these fixes are as you intended.

### CODE REVIEW

File: python/opdb/models.py [hunk 1]
```diff
@@ -453,6 +453,7 @@ class pfs_design(Base):
     is_obsolete = Column(Boolean)
 
     tiles = relation(tile, backref=backref('pfs_design'))
+    pfs_design_agcs = relation('pfs_design_agc', back_populates='pfs_design')
```
Comment (kiyoyabe) [inline python/opdb/models.py:457]:
```suggestion
pfs_design_fibers = relation('pfs_design_fiber', back_populates='pfs_design')

```

Comment (kiyoyabe) [reply python/opdb/models.py:457]: Please add this in the same reason.
Also could you add the following line in pfs_design_fiber class?
`pfs_design = relation('pfs_design', back_populates='pfs_design_fibers')`

**Review by kiyoyabe — CHANGES_REQUESTED** [review]
I just added comments and suggestions. Now I realize that `pfs_design` & `pfs_config`-related tables may be largely changed in `INSTRM-1352`, so I will take care of the relation again in that ticket.

File: python/opdb/models.py (hunk 1 above)
Comment (michitaro) [reply python/opdb/models.py:458]: I added the relation.

**Pull request merged**